toolchain/check/diagnostic_emitter.h

```diff
@@ -51,14 +51,20 @@ class DiagnosticEmitter : public DiagnosticEmitterBase {
   auto ConvertLocImpl(SemIR::LocId loc_id, bool is_token_only,
                       ContextFnT context_fn) const -> Diagnostics::ConvertedLoc;
 
-  // Converts a node_id corresponding to a specific sem_ir to a diagnostic
-  // location.
+  // Converts an `absolute_node_id` import from C++ or corresponding to a
+  // specific sem_ir to a diagnostic location.
```

Comment:
I'm having trouble parsing this sentence with "or ... to a ... to a", maybe:

```suggestion
  // Converts an `absolute_node_id` in either a Carbon file or C++ import to a
  // diagnostic location.
```

Reply:
Done.